src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -501,6 +510,7 @@ private EmailWrapper generateFeedbackSessionClosedEmail(CourseAttributes course,
                 "${courseId}", SanitizationHelper.sanitizeForHtml(course.getId()),
                 "${feedbackSessionName}", SanitizationHelper.sanitizeForHtml(session.getFeedbackSessionName()),
                 "${deadline}", SanitizationHelper.sanitizeForHtml(TimeHelper.formatTime12H(session.getEndTime())),
+                "${coOwnersEmails}", generateCoOwnersEmailsLine(generateCoOwnersList(course.getId())),
```

Comment:
You might also want to do the same for `generateFeedbackSessionClosedEmail` since it is generated in batch.